.github/workflows/release.yml

```diff
@@ -132,7 +132,7 @@ jobs:
             --user ${{ env.NEXUS_DEPLOY_USER }}
 
       - name: Promote artifacts
-        if: ${{ inputs.promote_artifacts == true }}
+        # if: ${{ inputs.promote_artifacts == true }}
```

Comment:
Wasn't the promote artifacts step suppose to switch position with the upload do Box?

Reply:
I am not sure if we need to promote artefacts at release bundle step

Reply:
We are not promoting artefacts while release bundle creation.

Reply:
Why?